rholang/src/test/scala/coop/rchain/rholang/interpreter/NormalizeTest.scala

```diff
@@ -115,7 +143,9 @@ class CollectMatcherSpec extends FlatSpec with Matchers {
       ProcNormalizeMatcher.normalizeMatch[Coeval](tuple, inputs).value
     }
   }
-
+  "Tuple" should "sort elements" in {
```

Comment:
we're lucky to have the `Sortable` typeclass - we can now property-test easily.

We should add a test to SortTest similar to "sort so that unequal terms have unequal scores and the other way around", that will test the following:
 - for every pair of sortables: their sortResult.score-s differ iff their sortResult.term-s differ
 - for every pair of sortables: whenever their sortResult.score-s differ or their sortResult.term-s differ, the input sortables themselves differ.

Reply:
Done